### PR TITLE
Closes about-page-view-creation-#17

### LINKED ISSUE #17: About page view creation

### PR DESCRIPTION
**Pull request by Snieezy**
closes #17

### CODE REVIEW

**Comment by hav3loc** [comment]
Verified, ready to be checked.

File: spec/helpers/about_helper_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# describe AboutHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe AboutHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (hav3loc) [inline spec/helpers/about_helper_spec.rb]: This helper doesn't really do anything.

File: app/views/layouts/application.html.slim [hunk 2]
```diff
@@ -1,10 +1,12 @@
 doctype html
 html
   head
-    title
-      | BoardGames
+    meta content=("text/html; charset=UTF-8") http-equiv="Content-Type"
```
Comment (hav3loc) [inline app/views/layouts/application.html.slim]: Shouldn't it be just part of header partial?

**Comment by hav3loc** [comment]
We need to get rid of all the files that are already part of master (layouts) as they are not relevant to this issue.

File: config/routes.rb [hunk 3]
```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: "home#index"
+  get "about" => "about#show"
```
Comment (hav3loc) [inline config/routes.rb]: I think it should be:
get 'about', to: 'about#show'

File: config/routes.rb [hunk 4]
```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: "home#index"
+  get 'about' => 'about#show'
```
Comment (hav3loc) [inline config/routes.rb]: I meant it should be:
`get "about", to: "about#show"` instead of `get "about" => "about#show"`
sorry for the confusion

**Comment by hav3loc** [comment]
Everything looks good +1

File: app/controllers/about_controller.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+class AboutController < ApplicationController
```
Comment (gotar) [inline app/controllers/about_controller.rb]: you can name those controller - StaticPages, and they you can have those
```
def about
  render ....
end
```

more generic way and will fit to other static pages if needed later

**Comment by gotar** [comment]
Squeeze to single commit and you can merge

**Pull request merged**